eclipsePlugin/test/de/tobject/findbugs/reporter/ThrottledProgressMonitorTest.java

```diff
@@ -0,0 +1,52 @@
+package de.tobject.findbugs.reporter;
```

Comment:
OK Got it. Then I will fix #160 first, and add a new project to add CLI entry for JUnit tests.

> this is how Eclipse plugins are structured and deployed

I see.

Even for me, a Maven hacker, it's natural to put test cases into the same project of production code (`src/main/java` and `src/test/java`). I didn't know the culture in Eclipse plugin community. I'll learn from you and that community.

Thanks again for your review!